padertorch/ops/mappings.py

```diff
@@ -5,11 +5,34 @@
 
 __all__ = [
     'ACTIVATION_FN_MAP',
-    'POOLING_FN_MAP',
-    'DTYPE_MAP',
 ]
 
-ACTIVATION_FN_MAP = Dispatcher(
+class CallableDispatcher(Dispatcher):
+    """
+       Is basically a dict for non-callable inputs
+       with a better error message on key error.
+       If the input is a callable it is returned.
+       >>> from padertorch.ops.mappings import CallableDispatcher
+       >>> d = CallableDispatcher(abc=1, bcd=2)
```

Comment:
Should one example work?

Reply:
this is an example taken from the original dispatcher since it behaves like a dictionary if it works, which is not interesting

Reply:
ok